course3/homework-string/my_string.h

```diff
@@ -1,6 +1,39 @@
-#pragma once
+#ifndef MY_STRING_H
```

Comment:
Nothing actually. The remnant of creating the project as new in qt creator I guess.